scripts/freebsd/vmtools.sh

```diff
@@ -44,7 +44,7 @@ vmware-iso|vmware-vmx)
     mdconfig -a -t vnode -f $HOME_DIR/freebsd.iso -u 0;
     mount -t cd9660 /dev/md0 /tmp/vmfusion;
     tar xzf /tmp/vmfusion/vmware-freebsd-tools.tar.gz -C /tmp/vmfusion-archive;
-    /tmp/vmfusion-archive/vmware-tools-distrib/vmware-install.pl --force-install;
+    /tmp/vmfusion-archive/vmware-tools-distrib/vmware-install.pl --default;
```

Comment:
Here are some easy lines to copy/pasta - https://github.com/chef/bento/blob/master/scripts/common/vmware.sh#L18-L23